core/commands/provide.go

```diff
@@ -0,0 +1,70 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+
+	cmds "github.com/ipfs/go-ipfs-cmds"
+	cmdenv "github.com/ipfs/kubo/core/commands/cmdenv"
+)
+
+const (
+	provideQuietOptionName = "quiet"
+)
+
+var ProvideCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Control provide opertaions.",
+		ShortDescription: `
+'ipfs provide' is a utility command used to control provide operations.
+`,
+	},
+
+	Subcommands: map[string]*cmds.Command{
+		"clear": provideClearCmd,
```

Comment:
:thought_balloon: in the future we may want to deprecate `ipfs routing provide` and `ipfs routing reprovide` have modern version at `ipfs provide stat`

Reply:
I can do that, but in a separate PR.

Reply:
We may want to deprecate `ipfs stats provide` and `ipfs stats reprovide`, but probably not `ipfs routing provide <cid>` since the command allows the user to advertise a provider record matching the cid to the Amino DHT.

Reply:
@guillaumemichel Want to create and issue?